.taskcluster.yml

```diff
@@ -250,21 +250,11 @@ tasks:
                   privileged: true
                 maxRunTime: 3600
                 image: "${taskboot_image}"
-                env:
-                  GIT_REPOSITORY: ${repository}
-                  GIT_REVISION: ${head_rev}
                 command:
-                  - taskboot
-                  - build
-                  - --image
-                  - mozilla/code-review
-                  - --tag
-                  - "${channel}"
-                  - --tag
-                  - "${head_rev}"
-                  - --write
-                  - /events.tar
-                  - events/docker/Dockerfile
+                  - /bin/sh
+                  - -lxce
+                  - "git clone --quiet ${repository} /code-review && cd /code-review && git checkout ${head_rev} -b build
+                    && backend/build.sh ${head_rev} ${head_branch} ${repository} ${channel}"
```

Comment:
I guess that's a bad copy-paste: you are building the backend instead of events here.

You need to call `events/build.sh` here.

Reply:
oops, good catch